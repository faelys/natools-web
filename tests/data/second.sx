(elements
   (main-text "\
<p>Data in the second page.</p>\n"
(render page-comments))
   (footer "<p>Special footer</p>\n")
   (title "Second Page"))
(tags (2 pages/news tags/topic2 tags/topic3))
(comments (backend comments 2) (tags comments) (flags closed))
(dates (published "2015-01-01T09:50:00-08:00"))
