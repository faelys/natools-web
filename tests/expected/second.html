<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
  <head>
    <title>Second Page</title>
  </head>
  <body>
<h1>Second Page</h1>
<p>Data in the second page.</p>
    <div class="sidebar">
<p>Tags: topic2 topic3</p>
    </div>
    <div class="footer">
<p>Special footer</p>
    </div>
  </body>
</html>
